lockbox-iosTests/AccountSettingPresenterSpec.swift

```diff
@@ -105,65 +106,11 @@ class AccountSettingPresenterSpec: QuickSpec {
                 }
 
                 describe("displayName") {
-                    describe("when the profileInfo has a display name") {
-                        let displayName = "meowskers"
-                        beforeEach {
-                            self.userInfoStore.profileStub.onNext(
-                                    ProfileInfo.Builder()
-                                            .displayName(displayName)
-                                            .email("blah@blah.com")
-                                            .build()
-                            )
-                        }
-
-                        it("tells the view to display the display name") {
-                            expect(self.view.displayNameObserver.events.last!.value.element).to(equal(displayName))
-                        }
-                    }
-
-                    describe("when the profileInfo does not have a display name") {
-                        let email = "meowskers@meow.com"
-                        beforeEach {
-                            self.userInfoStore.profileStub.onNext(
-                                    ProfileInfo.Builder()
-                                            .email(email)
-                                            .build()
-                            )
-                        }
-
-                        it("tells the view to display the email") {
-                            expect(self.view.displayNameObserver.events.last!.value.element).to(equal(email))
-                        }
-                    }
+                    // tricky to test because we can't construct FxAClient.Profile
```

Comment:
Maybe file an enhancement request with App Services?

Reply:
follow-up question in the meeting agenda for tomorrow